Assembly-CSharp.csproj

```diff
@@ -9,7 +9,7 @@
     <ProductVersion>10.0.20506</ProductVersion>
```

Comment:
Changes to this file seem to be a revert of previous changes.

I believe this project file should be excluded from version control, as Unity should be able to regenerate it. Exclude form the PR please.

Reply:
Ah, I didn't know that Unity auto regenerates this file. That's nice. It's been a pain with source control so far